app/assets/stylesheets/application.css.scss

```diff
@@ -89,8 +89,8 @@ input {
  margin-top: 5px;
 }
 
-.navbar-inverse {
-  margin-left: -125px;
+.navbar-fixed-bottom {
+  background: #263645 !important;
```

Comment:
Color literals like `#263645` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used